vendor/data/codelists/BEIS/fund_types.yml

```diff
@@ -5,3 +5,6 @@ data:
   - code: 2
     name: Global Challenges Research Fund
     short_name: GCRF
+  - code: 3
```

Comment:
Would the XML spec fail already as a result of this commit? (And if so, are you comfortable having a few "red" commits? ;) )

Reply:
Yes. 🙈

Reply:
(I mean yes it would fail; I still experience discomfort. But not much)

Reply:
Interesting! I should ask you to amend the commit title to say "RED:" :P (But I won't, I'll just let the idea float in the air ;) )